src/wwwroot/controllers/ReportsCtrl.js

```diff
@@ -28,7 +28,8 @@
 
     $scope.maxRateDailyLimitBar = 50;
     $scope.getDailyLimit = function() {
-      return $rootScope.accountLimits.daily && $rootScope.accountLimits.daily.limit;
+      
+      return $rootScope.accountLimits.daily && $rootScope.accountLimits.daily.limit?$rootScope.accountLimits.daily.limit:0;
```

Comment:
It is wrong, it is not the same _limit of zero_ than _no limits_

Reply:
I' will update it. Thanks